ansible/roles_ocp_workloads/ocp4_workload_example/tests/inventory

```diff
@@ -0,0 +1,2 @@
+localhost
```

Comment:
Do these tests actually work? Otherwise they're just cluttering things up...

Reply:
It's default galaxy stuff.

Reply:
deleting